audioswitch/lint-baseline.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<issues format="6" by="lint 7.4.2" type="baseline" client="gradle" dependencies="false" name="AGP (7.4.2)" variant="all" version="7.4.2">
+
+    <issue
+        id="MissingPermission"
+        message="Call requires permission which may be rejected by user: code should explicitly check to see if permission is available (with `checkPermission`) or explicitly handle a potential `SecurityException`"
+        errorLine1="    override val name: String = device.name ?: DEFAULT_DEVICE_NAME,"
+        errorLine2="                                ~~~~~~~~~~~">
+        <location
+            file="src/main/java/com/twilio/audioswitch/android/BluetoothDeviceWrapperImpl.kt"
+            line="9"
+            column="33"/>
+    </issue>
+
+    <issue
```

Comment:
what's the difference between this issue and what's on line 4?

Reply:
no idea.. I used the `gradlew updateLintBaseline` after I fixed all the real issues.